### PR TITLE
integrate RHOSAK TC for new UI changes

### PR DESCRIPTION
**Pull request by bdattoma**
there have been UI changes in a portion of RHOSAK flow. This PR wants to address it in the TC

### CODE REVIEW

**Review by lugi0 — APPROVED** [review]
lgtm

**Comment by bdattoma** [comment]
[Jenkins Job #372](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/372/)
TC succeeded

**Comment by bdattoma** [comment]
New test Run after few changes:  [Jenkins Job #372](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/378/)
_TC succeeded_

New changes include the PR #188

**Comment by bdattoma** [comment]
new run [JenkinsJob#393](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/393) after some changes applied (there was a new change in Log in page, I made the script more robust for changes in that page, I hope)
TC succeeded

**Review by lugi0 — DISMISSED** [review]
lgtm

File: tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot [hunk 1]
```diff
@@ -1,124 +1,130 @@
...
-  Close All Browsers
+    [Documentation]    Suite teardown keyword for RHOSAK component
+    Close All Browsers
 
 Kafka Test Setup
-  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  Wait for RHODS Dashboard to Load
-
+    [Documentation]    Test case setup keyword for RHOSAK component
```
Comment (jgarciao) [inline tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot:127]: Would you mind changing the [Documentation] tags to use the 3rd person?  You will find some examples in the [ODS-CI Robot Framework Style Guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit?usp=sharing)

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot (hunk 1 above)
Comment (bdattoma) [reply tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot]: done

**Review by jgarciao — APPROVED** [review]
LGTM. Thanks @bdattoma !

**Review by lugi0 — APPROVED** [review]

**Pull request merged**